src/Simplex/Messaging/Agent.hs

```diff
@@ -573,16 +624,43 @@ processSMPTransmission c@AgentClient {subQ} (srv, rId, cmd) = do
           | internalPrevMsgHash /= receivedPrevMsgHash = MsgError MsgBadHash
           | otherwise = MsgError MsgDuplicate -- this case is not possible
 
-connectToSendQueue :: AgentMonad m => AgentClient -> SndQueue -> SenderPublicKey -> VerificationKey -> m ()
-connectToSendQueue c sq senderKey verifyKey = do
-  sendConfirmation c sq senderKey
+confirmQueue :: AgentMonad m => AgentClient -> SndQueue -> SenderPublicKey -> ConnInfo -> m ()
+confirmQueue c sq senderKey cInfo = do
+  sendConfirmation c sq senderKey cInfo
   withStore $ \st -> setSndQueueStatus st sq Confirmed
-  sendHello c sq verifyKey
-  withStore $ \st -> setSndQueueStatus st sq Active
 
-newSendQueue ::
+activateQueueInitiating :: AgentMonad m => AgentClient -> ConnId -> SndQueue -> VerificationKey -> Int -> Int -> m ()
+activateQueueInitiating c connId sndQ verifyKey initialDelaySec increaseDelayAfterSec = do
```

Comment:
why do we need initialDelaySec and increaseDelayAfterSec parameters and not have inside sendHello? The are always set to the same values

Reply:
TODO type delay timing

Reply:
done